hypha/apply/funds/templates/funds/includes/rendered_answers.html

```diff
@@ -44,3 +58,30 @@ <h5 class="text-base">{% trans "Organization name" %}</h5>
 <div class="rich-text rich-text--answers">
     {{ object.output_answers }}
 </div>
+
+<style type="text/css">
+    #translate-card-loading span {
+        width: 490px;
+    }
+
+    #translate-card-loading.htmx-indicator{
+        height: 0;
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+        align-content: center;
+        border-width: 0px;
+    }
+    #translate-card-loading.htmx-request.htmx-indicator{
+        height: 64px;
+        transition: height 0.25s ease-in;
+        margin-top: 0.5rem;
+        margin-bottom: 0.5rem;
+        border-width: 1px;
+    }
+
+    #translate-card-loading.htmx-request.htmx-indicator span {
+        display: inline-block;
+        height: 1rem;
+    }
+</style>
```

Comment:
Could we use tailwind classes for this instead?

Reply:
the `htmx-request` class gets added while the request is processing so I that bit of CSS will need to stay but I moved everything else to tailwind